### ISSUE TITLE
Disable SSLProtocol 

### ISSUE THREAD

**Issue by mtalero-20** [jcrodriguez-dis/moodle-mod_vpl#105]
Es posible deshabilitar protocolos de cifrado ssl desde la configuración en /etc/vpl/vpl-jail-system.conf
SSLProtocol all -TLSv1 -SSLv3

**Comment by jcrodriguez-dis** [comment]
No, at this moment there is no option at vpl-jail-system.conf to disable specific SSL protocols.
To change the supported protocols you modify the code and recompile the daemon.
Change the code at
https://github.com/jcrodriguez-dis/vpl-xmlrpc-jail/blob/V2.5.3/src/socket.h#L52
Change it by SSL_CTX_set_options(context, SSL_OP_NO_SSLv3|SSL_OP_NO_TLSv1|SSL_OP_NO_TLSv1_1);

This will be included in the next version.

Best regards,
Juan Carlos.

**Comment by jcrodriguez-dis** [comment]
Please note that this issue is related to the VPL-JAIL-SYSTEM, which can be found in the repository [https://github.com/jcrodriguez-dis/vpl-xmlrpc-jail](https://github.com/jcrodriguez-dis/vpl-xmlrpc-jail).

I'm glad to inform you that the issue has been successfully resolved by adding the options `SSL_CIPHER_LIST` and `SSL_CIPHER_SUITES` to the configuration file.

For more detailed information, please refer to the [documentation](https://vpl.dis.ulpgc.es/documentation/vpl-jail-system-3.0.0/configuration.html#network-parameters) of VPL-JAIL-SYSTEM.
